scripts/driver.requirements.txt

```diff
@@ -15,5 +15,5 @@ Jinja2 >= 2.10.1; python_version <  '3.10'
 Jinja2 >= 2.10.3; python_version >= '3.10'
 # Jinja2 >=2.10, <3.0 needs a separate package for type annotations
 types-Jinja2 >= 2.11.9
-jsonschema >= 3.2.0
-types-jsonschema >= 3.2.0
+jsonschema >= 3.2.0, < 4.18.*
```

Comment:
I don't think that's right: it forbids recent versions of jsonschema. If you have the latest version installed, you should be able to run our scripts,

The problem is a [recurring one](https://github.com/Mbed-TLS/mbedtls/issues/8250) due to jsonschema's dependencies. It goes away once prebuilt wheels for the latest version become available, which generally takes a few days after each release.

We offer a script `scripts/min_requirements.py` which installs the minimum required version of each package. That's what we run on the CI.

Reply:
Do we need recent versions though? There is no harm in locking the maximum version of a library that is a moving target, unti it reaches EOL or we need to utilise some of their newest features.